README.md

```diff
@@ -1,12 +1,38 @@
 # Probot: CODEOWNERS labeller
 
-> a GitHub App built with [Probot](https://github.com/probot/probot) that adds labels in Pull Requests based on contents of the CODEOWNERS.
+> a GitHub App built with [Probot](https://github.com/probot/probot) that adds labels in Pull Requests based on contents of the `codeowners-labeller.yml` file.
 
 ## Usage
 
 1. **[Install the app](https://github.com/apps/elastic-codeowners-labeller)**.
-2. Create a `CODEOWNERS` file in your repository, under the `.github` directory.
-3. Wait for new Pull Requests to be opened or synchronised.
+1. Create a `codeowners-labeller.yml` file in your repository, under the `.github` directory.
+1. Use your own labels as the keys in the YAML file.
```

Comment:
Should this list be sequential? (i.e. 1, 2, 3)

Reply:
The cool thing about markdown is that it preserves the order, so we do not have to change different lines when reordering an item, or adding/deleting one. I discovered this a few days ago :)

Reply:
Ah, neat! I didn't know that. :)